src/Models/ContainerDeviceBase.ts

```diff
@@ -123,12 +123,11 @@ export abstract class ContainerDeviceBase implements Device {
     // Check remote
     const isRemote = RemoteExtension.isRemote(this.extensionContext);
     if (!isRemote) {
-      const res = await utils.askAndOpenInRemote(
+      await utils.askAndOpenInRemote(
           OperationType.Compile, this.channel, this.telemetryContext);
-      if (!res) {
-        return false;
-      }
+      return false;
```

Comment:
why always return false here? Does caller consume the return boolean value?

Reply:
The code below return true.
Here when not in remote, the compilation will fail and thus return false.